src/plugins/vis_type_vega/public/components/vega_help_menu.tsx

```diff
@@ -34,13 +34,17 @@ import React, { useCallback, useState } from 'react';
 import { EuiButtonIcon, EuiContextMenuPanel, EuiContextMenuItem, EuiPopover } from '@elastic/eui';
 import { FormattedMessage } from '@osd/i18n/react';
 import { i18n } from '@osd/i18n';
+import { useOpenSearchDashboards } from '../../../opensearch_dashboards_react/public';
 
 function VegaHelpMenu() {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
   const onButtonClick = useCallback(() => setIsPopoverOpen(!isPopoverOpen), [isPopoverOpen]);
 
   const closePopover = useCallback(() => setIsPopoverOpen(false), []);
 
+  const opensearchDashboards = useOpenSearchDashboards();
+  const vegaHelpDoc = opensearchDashboards.services.docLinks!.links.noDocumentation.vega;
```

Comment:
Nit: can we reduce this to one line since we don't use the `opensearchDashboards` variable anywhere else?

Reply:
will change.